scopes/harmony/application/app-build-result.ts

```diff
@@ -12,4 +12,9 @@ export interface AppBuildResult {
    * warnings thrown during the build process.
    */
   warnings?: string[];
+
+  /**
+   * metadata to persist.
+   */
+  metadata?: Record<string, string | undefined>;
```

Comment:
not sure this should be string, string, maybe 
string, any (you don't know which metadata will be added)

Reply:
We'll probably need to change the original `metadata`, which is `<string, Serializable>`. For now, I'll only change this and probably will need to add ts-ignore.

Reply:
this was related to the original `metadata` from the build-task which used "Serializable". I'm changing it to use `any`.